st2tests/integration/mistral/test_filters.py

```diff
@@ -0,0 +1,373 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import yaml
+
+from integration.mistral import base
+
+REGEX_SEARCH_STR = "Your address is 567 Elsewhere Dr. My address is 123 Somewhere Ave."
+REGEX_SEARCH_STR_2 = "567 Elsewhere Dr is your address. My address is 123 Somewhere Ave."
+REGEX_SEARCH_STR_3 = "No address to be found here! Well, maybe 127.0.0.1"
+
+
+class JsonEscapeFiltersTest(base.TestWorkflowExecution):
+
+    def test_json_escape(self):
+        breaking_str = 'This text """ breaks JSON'
+        inputs = {'input_str': breaking_str}
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-json_escape', parameters=inputs
+        )
```

Comment:
Please be consistent with the indentation. For the tests, either make all of them

```
execution = self._execute_workflow('examples...',
                                   parameters...
``` 

OR

```
execution = self._execute_workflow(
    'examples...', parameters...
)
```

OR

```
execution = self._execute_workflow(
    'examples...',
    parameters...
)
```